src/themes/darkTheme.ts

```diff
@@ -48,6 +41,16 @@ const darkTheme = createMuiTheme({
         color: Color.BlueMegaLight,
       },
     },
+    MuiFormLabel: {
+      root: {
+        '&.Mui-disabled': {
+          color: Color.Black,
+        },
+        '&.Mui-focused': {
```

Comment:
hva betyr det egentlig at disse er prefixed med `&`?

Reply:
Det betyr "denne regelen gjelder for de som har klassene .MuiFormLabel-root og .Mui-focused"